### PR TITLE
completed 12-sorter

### PR DESCRIPTION
**Pull request by goelvatsal**

### CODE REVIEW

File: section3/set11/exercises/12-sorter/main.go [hunk 1]
```diff
@@ -53,41 +53,42 @@ import (
...
-	}
 	fmt.Printf("%v\n", a)
 
 	// add bubble sorting program
-	for i := 0; i < 5; i++ {
-		for j := 0; j < 5-i; j++ {
+	for i := 0; i < len(args); i++ {
+		for j := 0; j < len(args); j++ {
+			if j == 4 {
+				break
```
Comment (goel4607) [inline section3/set11/exercises/12-sorter/main.go]: I do not see a need for this check and the break statement

File: section3/set11/exercises/12-sorter/main.go [hunk 2]
```diff
@@ -53,41 +53,42 @@ import (
...
-	for i := 1; i < 6; i++ {
-		a[i] = arrN[i]
-	}
 	fmt.Printf("%v\n", a)
 
 	// add bubble sorting program
-	for i := 0; i < 5; i++ {
-		for j := 0; j < 5-i; j++ {
+	for i := 0; i < len(args); i++ {
+		for j := 0; j < len(args); j++ {
```
Comment (goel4607) [inline section3/set11/exercises/12-sorter/main.go]: change the check here

File: section3/set11/exercises/12-sorter/main.go (hunk 1 above)
Comment (goelvatsal) [reply section3/set11/exercises/12-sorter/main.go]: removed the check and break statement

File: section3/set11/exercises/12-sorter/main.go (hunk 2 above)
Comment (goelvatsal) [reply section3/set11/exercises/12-sorter/main.go]: fixed the check to i < 4 for the second loop so the previous check and break statement can be deleted

**Pull request merged**